src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -93,4 +96,14 @@ private Optional<Set<Tag>> parseTagsForEdit(Collection<String> tags) throws Pars
         return Optional.of(ParserUtil.parseTags(tagSet));
     }
 
+    private Optional<Set<Days>> parseDaysAvailableForEdit(Collection<String> days) throws ParseException {
+        assert days != null;
+
+        if (days.isEmpty()) {
+            return Optional.empty();
+        }
+        Collection<String> daySet = days.size() == 1 && days.contains("") ? Collections.emptySet() : days;
```

Comment:
Very very tiny nit, but do you think that wrapping the boolean condition in parenthesis will enhance readability? (days.size()==1 && days.contains(""))?